### DISCUSSION TITLE
CloseModal() throws an error and freezes window

### DISCUSSION THREAD

**Discussion by rickjm540** [wire-elements/modal#187] [Q&A]
Following your example, everythign works, but when I try to add     

<button wire:click="$emit('closeModal')">Close Modal</button>

and click the button, it throws an error in the console and freezes. Here's the error.
![Screen Shot 2022-02-28 at 2 46 08 PM](https://user-images.githubusercontent.com/1508104/156056943-fa15c70a-4c32-4b6f-a3ef-eab398bdaf06.png)
![Screen Shot 2022-02-28 at 2 45 53 PM](https://user-images.githubusercontent.com/1508104/156056948-39da5319-bfda-4763-ae32-adf666c16cb8.png)

**Comment by Ceepster14** [comment]
Can I add my voice to this. I'm experiencing exactly the same issue. Laravel 9.5.1 / Alpine.js 3.9.5 / Chrome 99.0.4844.84. I'm also triggering the modal from within a Rappasoft LiveWire Table if that helps shed any light.

**Reply by Ceepster14** [reply]
OK - Solved this. I didn't realise I already had Alpine included in my app.js file but was also bringing it in from a CDN. Removed that and now  closeModal() functions correctly.

**Reply by rickjm540** [reply]
Oh! That was my problem too!
Thank you for adding a reply to this :)

**Comment by DZ4-Miguel** [comment]
Hello, I just have the same error. Can someone please explain this again in more detail?
